Add ability to create test runs in Azure DevOps (#2931)
vividus-framework/vividus: docs/modules/integrations/partials/test-case-types.adoc

== Test Case Types

> This separation of automated and manual test cases by using special keywords is specific to a VIVIDUS.

Reviewer: ```suggestion
This separation of automated and manual test cases by using special keywords is specific to VIVIDUS.
```

=== Automated Test Cases

> Any scenario that doesn't correspons to the <<_manual_test_cases, manual test case rules>> is considered as automated test case

Reviewer 2: ```suggestion
Any scenario that doesn't correspond to the <<_manual_test_cases, manual test case rules>> is considered as automated test case.
```

Examples:
|locator                        |
|By.linkUrl(#ElementId)         |
|By.linkUrlPart(Element)        |
|By.linkText(Link to an element)|
----

.AutomatedWithManualPart.story
[source,gherkin]
----
Scenario: Verify link
Meta: @testCaseId TEST-566

!-- Step: Open main app page
!-- Step: Wait for page with title is loaded
!-- Step: Verify number of links
!-- Data: * link url is '#ElementId'
!-- * link url part is 'Element'
!-- * link text is 'Link to an element'
!-- Result: The number of links for all locators is equal to 1

Given I am on a page with the URL '${vividus-test-site-url}/links.html'
When I wait until the page has the title 'Links'
Then number of elements found by `<locator>` is equal to `1`
Examples:
|locator                        |
|By.linkUrl(#ElementId)         |
|By.linkUrlPart(Element)        |
|By.linkText(Link to an element)|
----

=== Manual Test Cases

* Scenario is considered as `Manual Test Case` if all of its lines start with `!--` prefix.
* The `Manual Test Case` step must start with `Step:` used to specify action to perform and can optionally contain `Data:` and `Result:` for specifying action data and action expected result respectively.

> * The `Manual Test Case` step parts are allowed to have multilines.

Reviewer 2: ```suggestion
* The `Manual Test Case` step parts are allowed to have multiple lines.
```
